spec/models/redirection_spec.rb

```diff
@@ -33,11 +32,10 @@
 
   describe "#previous_url" do
     it "returns the url of the previous referenced redirection" do
-      gabe = create(:redirection)
-      edward = create(:redirection, next: gabe)
-      gabe.update(next: edward)
+      gabe = Redirection.find_by!(slug: "gabe")
+      edward = Redirection.find_by!(slug: "edward")
 
-      expect(gabe.previous_url).to eq(edward.url)
+      expect(edward.previous_url).to eq(gabe.url)
```

Comment:
All looks good, but may be worth considering a spec where the ring has more than two members, just in case.